packages/cms-cli/commands/init.js

```diff
@@ -20,18 +24,58 @@ const { addLoggerOptions, setLogLevel } = require('../lib/commonOpts');
 const { logDebugInfo } = require('../lib/debugInfo');
 
 const COMMAND_NAME = 'init';
+const HS_AUTH_OAUTH_COMMAND = 'hs auth oauth2';
 
 const promptUser = async promptConfig => {
   const prompt = inquirer.createPromptModule();
   return prompt(promptConfig);
 };
 
+const oauthConfigSetup = () => {
+  try {
+    createEmptyConfigFile();
+    const authProcess = spawn(HS_AUTH_OAUTH_COMMAND, {
```

Comment:
Instead of spawning a new process can we refactor the code for the oauth command so that we can do it as part of the same process?

Reply:
Done.